packages/backend-test-utils/package.json

```diff
@@ -53,8 +53,8 @@
     "@backstage/plugin-auth-node": "workspace:^",
     "@backstage/plugin-events-node": "workspace:^",
     "@backstage/types": "workspace:^",
-    "@keyv/memcache": "^1.3.5",
-    "@keyv/redis": "^2.5.3",
+    "@keyv/memcache": "^2.0.1",
+    "@keyv/redis": "^4.0.1",
     "@types/express": "^4.17.6",
```

Comment:
are these deps actually used? could they be removed instead? I'd assume some code changes are needed otherwise

Reply:
There are used here:
- memcache: https://github.com/backstage/backstage/blob/master/packages/backend-test-utils/src/cache/memcache.ts
- redis: https://github.com/backstage/backstage/blob/master/packages/backend-test-utils/src/cache/redis.ts

I don't think any code changes are needed since they are only used to create a test connection without any additional options. And there are tests covering these test connections like: https://github.com/backstage/backstage/blob/master/packages/backend-test-utils/src/cache/redis.test.ts